tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

```diff
@@ -173,90 +182,108 @@ func testSetup(ctx resource.Context) (err error) {
-			Subsets: []echo.SubsetConfig{
-				{
-					Annotations: map[echo.Annotation]*echo.AnnotationValue{
-						echo.SidecarBootstrapOverride: {
-							Value: sdBootstrapConfigMap,
+
+	builder := echoboot.NewBuilder(ctx)
+	for _, cls := range ctx.Clusters() {
+		clName := cls.Name()
+		builder.
+			With(nil, echo.Config{
+				Service:   fmt.Sprintf("clt-%s", clName),
+				Cluster:   cls,
+				Namespace: getEchoNamespaceInstance(),
+				Subsets: []echo.SubsetConfig{
+					{
+						Annotations: map[echo.Annotation]*echo.AnnotationValue{
+							echo.SidecarBootstrapOverride: {
+								Value: sdBootstrapConfigMap,
+							},
 						},
 					},
+				}}).
+			With(nil, echo.Config{
+				Service:   "srv",
+				Cluster:   cls,
+				Namespace: getEchoNamespaceInstance(),
+				Ports: []echo.Port{
+					{
+						Name:     "grpc",
+						Protocol: protocol.GRPC,
+						// We use a port > 1024 to not require root
+						InstancePort: 7070,
+					},
+					{
+						Name:     "http",
+						Protocol: protocol.HTTP,
+						// We use a port > 1024 to not require root
+						InstancePort: 8888,
+					},
+					{
+						Name:     "tcp",
+						Protocol: protocol.TCP,
+						// We use a port > 1024 to not require root
+						InstancePort: 9000,
+					},
 				},
-			}}).
-		With(&srv, echo.Config{
-			Service:   "srv",
-			Namespace: getEchoNamespaceInstance(),
-			Ports: []echo.Port{
-				{
-					Name:     "grpc",
-					Protocol: protocol.GRPC,
-					// We use a port > 1024 to not require root
-					InstancePort: 7070,
-				},
-				{
-					Name:     "http",
-					Protocol: protocol.HTTP,
-					// We use a port > 1024 to not require root
-					InstancePort: 8888,
-				},
-				{
-					Name:     "tcp",
-					Protocol: protocol.TCP,
-					// We use a port > 1024 to not require root
-					InstancePort: 9000,
-				},
-			},
-			Subsets: []echo.SubsetConfig{
-				{
-					Annotations: map[echo.Annotation]*echo.AnnotationValue{
-						echo.SidecarBootstrapOverride: {
-							Value: sdBootstrapConfigMap,
+				Subsets: []echo.SubsetConfig{
+					{
+						Annotations: map[echo.Annotation]*echo.AnnotationValue{
+							echo.SidecarBootstrapOverride: {
+								Value: sdBootstrapConfigMap,
+							},
 						},
 					},
-				},
-			}}).
-		Build()
+				}}).
+			Build()
+	}
+	echos, err := builder.Build()
 	if err != nil {
 		return
 	}
+	clt = echos.Match(echo.ServicePrefix("clt"))
+	srv = echos.Match(echo.Service("srv"))
 	return nil
 }
 
 // send both a grpc and http requests (http with forced tracing).
 func sendTraffic(t *testing.T) error {
 	t.Helper()
+	//  All server instance have same names, so setting target as srv[0].
+	// Sending the number of total request same as number of servers, so that load balancing gets a chance to send request to all the clusters.
 	grpcOpts := echo.CallOptions{
-		Target:   srv,
+		Target:   srv[0],
 		PortName: "grpc",
-		Count:    1,
-	}
-	if _, err := clt.Call(grpcOpts); err != nil {
-		return err
+		Count:    3 * len(srv),
```

Comment:
just curious: why 3 ?

Reply:
Something more than 1 and will guarantee for sure we have cross cluster traffic..
@stevenctl : do we have a recommendation for the multiplies here?

Reply:
I've been using 5, but I'm also checking for even distribution and stuff so 3 should be plenty sufficient if you just want at least one hit-per-cluster

Reply:
can we make this a const somewhere with a comment explaining the purpose and function?  it is sprinkled in a lot of places and i'm worried three months from now it will be mysterious.